torchvision/transforms/v2/functional/_geometry.py

```diff
@@ -62,6 +75,16 @@ def _horizontal_flip_image_pil(image: PIL.Image.Image) -> PIL.Image.Image:
     return _FP.hflip(image)
 
 
+def _horizontal_flip_image_cvcuda(image: "cvcuda.Tensor") -> "cvcuda.Tensor":
```

Comment:
Maybe a bit of a nitpick, but could we rename the function to `_horizontal_flip_cvcuda`, CV-CUDA only operates on one datatype so the extra "image" in the funcname does not add value IMO. Removing it also mirrors the `cvcuda_to_tensor` and `tensor_to_cvcuda` functions